website/docs/components/side-nav/partials/code/how-to-use.md

```diff
@@ -406,7 +406,13 @@ When the Side Nav is paired with the [`Hds::AppHeader`](/components/app-header)
 
 ### Responsiveness
 
-As mentioned above, the full-fledged `Hds::SideNav` component is "responsive" by default:
+!!! Info
```

Comment:
I've read multiple times the content, and I _think_ we don't need this banner here. Is not adding much and is not in the "right" place (is under the "full-fledged" section. Since there is no specific section about a sidenav build the using the atomic blocks, but at the same responsive (which is a bit of an edge case) I think we can avoid adding even more content, and leave just the changes suggested above. What do you think?

Reply:
Thanks, I struggled with this one a bit. I wanted to add something more prominent such as a banner but also wasn't sure of the best place to add one so I put it here as this seemed like an ok related spot. If you really don't think it adds much though I can just delete it.

Reply:
I'll just go ahead and delete